tests/FileReaderTest.php

```diff
@@ -41,7 +41,6 @@ public function testCreateSingleton(): void
     {
         $instance = FileReader::createSingleton();
 
-        static::assertInstanceOf(\PhpCsFixer\FileReader::class, $instance);
```

Comment:
with this, we may get more user-friendly failure message. any value in dropping it?

Reply:
it is impossible to be of any other type because of the return type added to `createSingleton` and the strict declaration

Reply:
here is another one in a PR https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5900/files#diff-ca1170c2b42d753009cd3a773fe200b4d4bfb4840e0930741eafc5e860149feaL31